ms2deepscore/train_new_model/data_generators.py

```diff
@@ -156,7 +156,7 @@ def _data_augmentation_spectrum(self, spectrum_tensor):
             # TODO: Factor out function with documentation + example?
             
             indices_select = torch.where((spectrum_tensor > 0)
-                                         & (spectrum_tensor < self.model_settings.augment_removal_max))[0]
+                                         & (spectrum_tensor < self.model_settings.augment_removal_intensity))[0]
```

Comment:
What is the difference here? Because the augment_removal_max is also still in the settings

Reply:
@florian-huber

Reply:
Only peaks with intensities < `augment_removal_intensity` are selected for potential removal.
Out of those, a fraction of 0 to `removal_max` can be removed.

Reply:
Ah so this was a bug before?